maintainers/maintainer-list.nix

```diff
@@ -14635,4 +14635,11 @@
     github = "snpschaaf";
     githubId = 105843013;
   };
+  jsimonetti = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.